packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -196,6 +196,11 @@ export class ListItem
     this.calciteInternalListItemSelect.emit();
   }
 
+  /**
+   * When `true`, the component's content appears inactive.
```

Comment:
+@geospatialem @DitwanP for doc review

Reply:
@geospatialem @DitwanP i'll merge now. We can update the doc post install if we need to.

Reply:
The doc LGTM! We can add a new sample to showcase the behavior for folks once `2.13.0` is live for additional context as well. ⚡